### DISCUSSION TITLE
Reopen pullreguest: Pull reguest doesn't see changes in dev branch

### DISCUSSION THREAD

**Discussion by sbeach92** [community/community#78134] [New to GitHub]
I have made pullreguest:
https://github.com/openwrt/openwrt/pull/13571/commits

which is linked to:
https://github.com/sbeach92/openwrt/tree/ASUS-LYRA-MINI-MAP1300

I assume I got that closed state as I was trying to revert unwanted merge. Now changes in my own repo is not shown and I can't reopen that pullreguest.

How can I get my code back to pullreguest?

**Comment by queenofcorgis** [comment]
Hi there, 

I'm not sure which settings are on this repo, but you likely will have to open a new pull request on that repo. For more details on how to use a specific repository on GitHub, the best way is to contact the maintainers of that software:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
